modules/social_features/social_profile/modules/social_profile_preview/src/Service/SocialProfilePreviewHelper.php

```diff
@@ -79,7 +79,12 @@ public function alter(
         $attributes = $attributes->toArray();
       }
 
-      $attributes['class'][] = 'preview-popup-link';
+      if ($attributes['loading']) {
```

Comment:
@Agamich-FE We have PHP error related with this change:
```
Warning(php): <em class="placeholder">Warning</em>: Undefined array key "loading" in <em class="placeholder">Drupal\social_profile_preview\Service\SocialProfilePreviewHelper-&gt;alter()<
```

Probably you can try to add check before, with `isset()` or `array_key_exists()`

Reply:
@ribel, I've added a fix. Thanks.